src/formats.rs

```diff
@@ -2957,6 +2957,13 @@ formats! {
     extension = "wmf"
     kind = Image
 
+    format = EnhancedMetafile
```

Comment:
Looks good to me, but should be sorted by format ident ascending, here in W variants.